src/branchingpriority.jl

```diff
@@ -0,0 +1,18 @@
+struct VarBranchingPriority <: MOI.AbstractVariableAttribute end
+
+branchingpriority!(model, x::JuMP.VariableRef, branching_priority::Int) = MOI.set(
```

Comment:
Does it work if one wants to set the branching priority of a collection of variables ?

For instance : 
```julia
@variable(m, x[1:10])
branchingpriority!(x, 2)
```

Reply:
Not yet, but it's simple to make that happen. Should `branchingpriority(model, x)` ("get" method) return a single `Int`?

Reply:
Good question. I think we'll have different branching priorities in a given collection. Perhaps, the get method should have a behaviour similar to `JuMP.value` ?